src/utils.ts

```diff
@@ -86,3 +90,49 @@ export const createComments = async (
     })
   }
 }
+
+export const updateLabels = async (
+  token: string,
+  owner: string,
+  repo: string,
+  pulls: Pull[],
+  successfulPulls: Pull[],
+  customSuccessLabel: string,
+  customFailureLabel: string
+): Promise<void> => {
+  info('update label')
+
+  const octokit = getOctokit(token)
```

Comment:
nit: we should only initialize this once and pass it as dependency.

Reply:
I will try to do this in the `update-comments` branch... :D